Client/ClientRoom.cpp

```diff
@@ -8,6 +8,7 @@
 /// @file Client/ClientRoom.cpp
 
 #include "ClientRoom.hpp"
+#include <csignal>
 #include <functional>
 #include "Error/Error.hpp"
 #include "GraphicECS/SFML/Components/ActionQueueComponent.hpp"
```

Comment:
Not in a good order ?

Reply:
:eyes:

Reply:
CI action is not okay with that, so sob sob